plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/PlayAssetDeliveryTestHelper.java

```diff
@@ -46,4 +51,136 @@ public static Dictionary createAssetPackLocationsDictionary() {
     testDict.put("location2", innerDict2);
     return testDict;
   }
+
```

Comment:
I'm thinking about if these anonymous class could be refactored. I looked into Mockito and I can't seem to find the feature that helps me mock a method with side effects.

Reply:
You can use doAnswer to mock a method to add side effects (4 in baeldung.com/mockito-void-methods#answering-a-call-to-void). I would definitely use that approach over anonymous classes to reduce the boilerplate code that you'll need to write for each type of task.

Reply:
Adapted the doAnswer approach. Now the helper class looks much cleaner!